RubeesEat/Startup.cs

```diff
@@ -16,11 +16,6 @@ public void ConfigureServices(IServiceCollection services)
     {
         services.AddRazorPages();
 
-        var todoRepository = new InMemoryTodoRepository();
-        todoRepository.Add(TodoItem.Create("My first todo"));
-
-        services.AddSingleton<ITodoRepository>(todoRepository);
-
         ConfigureServicesEAT(services);

```

Comment:
I would suggest inlining the `ConfigureServicesEAT` method into `ConfigureServices`. There is no longer a need to have a separate method.